docsource/source/usage.rst

```diff
@@ -155,6 +155,20 @@ The ``export_csvs()`` method generates a zip file, containing :doc:`clips_csv` a
     
     build.export_csvs("build_csvs.zip")
 
+The ``bsb.pdf_instructions()`` function creates pdf instructions for manual assembly of the build in the lab. An example can be seen here_
+
+.. _here: https://github.com/LondonBiofoundry/basicsynbio/blob/master/docsource/source/literal_includes/export_pdf_example.pdf
+
+.. code-block::
+    
+    bsb.export_csvs(build)
+
+The ``bsb.export_echo_assembly(build)`` function creates echo lab automation instructions for the clips to assembly step of the build. Example outputs, :doc:`echo_clips_1_csv` and :doc:`echo_water_buffer_1_csv`
```

Comment:
The ``bsb.export_echo_assembly(build)`` function creates [Echo liquid handling instructions](https://www.mybeckman.uk/liquid-handlers/echo-525) to combine purified clip reactions with buffer and water, yielding assemblies for transformation. The following :doc:`echo_clips_1_csv` and :doc:`echo_water_buffer_1_csv` are example outputs.